src/client/components/robot/model.ts

```diff
@@ -1,4 +1,15 @@
+import throttle = require('lodash.throttle')
 import { observable } from 'mobx'
+import { runInAction } from 'mobx'
+
+type RobotModelOpts = {
```

Comment:
With the new `dataRates` property, the class RobotModel can no longer be used as an interface, so I added this for use in the constructor and `of()`